_posts/2017-08-30-5-myths-about-reading.markdown

```diff
@@ -0,0 +1,37 @@
+
+The first question thrown towards me was: What is the fundamental difference between a competent reader and an ineffective reader? An efficient reader has the right mindset towards reading. They know why they are reading and what they should expect out of the book. On the other hand, an ineffective reader has many misconceptions about reading. I will be discussing few of those misunderstandings in this blog.
+
+**1. You cannot read more than 1k words per minute**
+
+This is a misconception that many people have. Individuals with practice have achieved this reading speed and still understand the text. Reading is a skill that can be improved over time. Thinking that you can improve indefinitely would be foolish. There is a limit till where humans can read and comprehend the text simultaneously. After a certain threshold, we read from our unconscious mind, and our comprehension becomes very weak. Hence realizing your toppling point helps you improve your
+reading capabilities.
+
+**2. You can understand 100% of the text**
+
+You can never understand the complete text in one go. You may require multiple sitting to read the book. Even after multiple sessions, if you pick up the book after a period you could still get new insights from the book. This happens because over the period you have improved and had many changes in your perceptions. Hence one should never focus on getting 100% out of a book, rather concentrate on getting the main gist of the book.
+
+**3. You must read slowly to get better comprehension**
+
+Everyone has their reading speeds and comprehension level. Your reading speed doesn't justify your understanding of the book. It is a misconception that reading faster can reduce your comprehension of your text. Yes after you have reached your usual reading speed, comprehension level starts to decrease, but this doesn't mean that you should always read slowly and it will always help you in understanding the text better. Many also have a misconception that one should read slow when reading
+novels or poetry to appreciate it. This thought is not right; you would miss the rhythm of the text if you go too slow. Hence it is important that you understand your speed of reading does not relate to the comprehension of the text.
+
+**4. You should keep a dictionary with you while reading**
+
+Instant lookup of meaning distracts you from your reading flow. It acts as an obstruction in reading. Mostly you would understand the bigger meaning even if you don't understand the significance of that particular word. Rather you should highlight the words that you find challenging and revisit them once that reading session is over. This would help both in increasing the vocabulary and also improving your reading experience overall.
```

Comment:
give example how you can understand the complete phrase even if you don't understand the word's meaning

Reply:
increasing the vocabulary -> vocabulary is developed not increased